OpenRA.Mods.Common/Traits/Buildings/TransformsIntoAircraft.cs

```diff
@@ -63,7 +63,7 @@ IEnumerable<IOrderTargeter> IIssueOrder.Orders
 				if (!IsTraitDisabled)
 				{
 					yield return new EnterAlliedActorTargeter<BuildingInfo>("Enter", 5, AircraftCanEnter,
-					 target => Reservable.IsAvailableFor(target, self));
+						target => Reservable.IsAvailableFor(target, self));
```

Comment:
While you're at it, wouldn't it help readability to insert an empty line below?

Reply:
Done.